jetty/src/main/scala/org/http4s/server/jetty/JettyBuilder.scala

```diff
@@ -388,3 +393,15 @@ object JettyBuilder {
 }
 
 private final case class Mount[F[_]](f: (ServletContextHandler, Int, JettyBuilder[F]) => Unit)
+
+case class PrefixedService[F[_]](service: HttpRoutes[F], prefix: String)
+class PrefixedServices[F[_]] private (val services: List[PrefixedService[F]])
+
+object PrefixedServices {
+  def apply[F[_]](service: HttpRoutes[F], prefix: String) =
+    new PrefixedServices[F](List(PrefixedService(service, prefix)))
+
+  implicit def prefixedServicesSemigroup[F[_]]: Semigroup[PrefixedServices[F]] =
```

Comment:
I think this could even be a `Monoid` with an empty list of services.

I like to law check our instances, but we have no notion of equality for `HttpRoutes`.  But I feel pretty good about this being lawful.